### DISCUSSION TITLE
curl: (35) Recv failure: Operation timed outGetting Started

### DISCUSSION THREAD

**Discussion by Froches** [Homebrew/discussions#5115] [Getting started]
### Output of `brew config`

```shell
mac@Macs-MacBook-Air ~ % brew config
zsh: command not found: brew
```

### Output of `brew doctor`

```shell
mac@Macs-MacBook-Air ~ % brew doctor
zsh: command not found: brew
```

### Description of issue

I had Homebrew installed on my MacBook Air 13" with OS Big Sur 11.7.10.
Each time I tried to brew install something, I got an error when it got to "openssl1@3" and it would be unsuccessful after that.
This kept happening for days and i tried updating, upgrading, etc to no avail. When I passed the issue to GPT for possible solutions, it recommended that i uninstall and reinstall Homebrew to see if that might work. Since then, I have been unable to reinstall it with any success.
Every time I try to install it, it will run for at least 5mins with no output and then say "curl: (35) Recv failure: Operation timed out"
I don't know what to do and I really need this to work.

**Comment by SMillerDev** [comment]
That seems like a problem with your connection to GitHub. Do you have the same issue with the installer package file?
